### PR TITLE
add the security lifecycle big picture diagram to getting started

### PR DESCRIPTION
**Pull request by em-c-rod**
Signed-off-by: Emily Rodriguez <ecrodriguez@mm279976-pc.lan>

### CODE REVIEW

**Comment by wdower** [comment]
Can you make the background on the image transparent? I think you can do that while you export from Powerpoint. 

You'd actually need to make two versions, one with white text and one with dark text, so that it works in both light and dark modes, and then have Vue serve one or the other depending on the light/dark toggle.

**Comment by Amndeep7** [comment]
I think it's fine for right now to just paste the graphic, but we should make sure to have some more text either above it or as a description for the graphic to explain it as an accessibility concern.

**Review by Amndeep7 — CHANGES_REQUESTED** [review]
I think it's fine for right now to just paste the graphic, but we should make sure to have some more text either above it or as a description for the graphic to explain it as an accessibility concern.

**Review by wdower — COMMENTED** [review]
Image works to start as we discussed, but can we get a higher-res export of it? Looks fuzzy to me on my monitor at 1920x1200

**Comment by ejaronne** [comment]
The Plan column seems a bit developing-of-STIGs-focused. Recommend change "Develop STIGs from SRGs" to "Choose, tailor, and create security guidance appropriate for your mission"

**Comment by aaronlippold** [comment]
Looks good with update rez

File: src/assets/data/resources.json [hunk 1]
```diff
@@ -11,6 +11,11 @@
         }
       ]
     },
+    { 
+      "name": "SAF Tooling at a Glance",
```
[the remark this answers is not in the document]
Comment (Amndeep7) [reply src/assets/data/resources.json:31]: I could go either way on this one.  Do either of @wdower or @em-c-rod feel strongly one way or the other?

**Comment by Amndeep7** [comment]
> The Plan column seems a bit developing-of-STIGs-focused. Recommend change "Develop STIGs from SRGs" to "Choose, tailor, and create security guidance appropriate for your mission"

@ejaronne This is just a copy paste of the slide graphic so making adjustments esp with that much additional text is hard.  For now we do have the bottom half of that column which mentions pre-existing benchmarks, etc.

**Review by Amndeep7 — APPROVED** [review]

**Pull request merged**